package.json

```diff
@@ -10,40 +10,52 @@
     "node": "7.7.1"
   },
   "dependencies": {
-    "babel-preset-es2015": "6.18.0",
-    "grunt": "1.0.1",
-    "grunt-angular-templates": "1.1.0",
-    "grunt-babel": "6.0.0",
-    "grunt-cache-bust": "1.4.1",
-    "grunt-cli": "1.2.0",
-    "grunt-contrib-clean": "1.0.0",
-    "grunt-contrib-concat": "1.0.1",
-    "grunt-contrib-copy": "1.0.0",
-    "grunt-contrib-cssmin": "1.0.2",
-    "grunt-contrib-uglify": "2.0.0",
-    "grunt-usemin": "3.1.1",
-    "load-grunt-tasks": "3.5.2"
+    "angular": "1.5.8",
+    "angular-cookies": "1.5.8",
+    "angular-local-storage": "0.5.0",
+    "angular-resource": "1.5.8",
+    "angular-route": "1.5.8",
+    "angular-sanitize": "1.5.8",
+    "angular-toarrayfilter": "1.0.1",
+    "angular-ui-bootstrap": "1.3.3",
+    "angular-ui-router": "0.4.2",
+    "bootstrap": "3.3.5",
+    "font-awesome": "4.4.0",
+    "hawk": "2.3.1",
+    "jquery": "2.1.3",
+    "jquery.scrollto": "2.1.0",
+    "js-yaml": "3.7.0",
+    "lodash": "3.10.1",
+    "mousetrap": "1.4.6",
+    "neutrino": "4.3.0",
+    "neutrino-lint-base": "4.2.0",
+    "neutrino-preset-karma": "4.2.0",
+    "neutrino-preset-react": "4.2.3",
+    "ngreact": "0.3.0",
+    "raw-loader": "0.5.1",
+    "react": "15.3.1",
+    "react-dom": "15.3.1",
+    "taskcluster-client": "1.6.3",
+    "webpack-md5-hash": "0.0.5"
   },
   "devDependencies": {
-    "browserify": "^13.1.1",
+    "angular-mocks": "1.5.8",
+    "enzyme": "2.7.1",
     "eslint": "3.8.0",
-    "grunt-eslint": "19.0.0",
-    "grunt-html-angular-validate": "0.5.8",
+    "istanbul-instrumenter-loader": "2.0.0",
     "jasmine-core": "2.5.2",
-    "karma": "1.3.0",
-    "karma-browserify": "^5.1.0",
-    "karma-coverage": "1.1.1",
+    "karma": "1.5.0",
+    "karma-coverage-istanbul-reporter": "0.2.3",
     "karma-firefox-launcher": "1.0.0",
-    "karma-jasmine": "1.0.2",
-    "karma-junit-reporter": "1.0.0",
-    "karma-ng-scenario": "0.1.0",
-    "minimist": "1.2.0",
-    "ngreact-test-utils": "^1.0.3",
-    "react": "^15.3.1",
-    "react-dom": "^15.4.1",
-    "watchify": "^3.8.0"
+    "karma-jasmine": "1.1.0",
+    "react-addons-test-utils": "15.3.1"
   },
   "scripts": {
-    "test": "karma start tests/ui/config/karma.conf.js --single-run --browsers Firefox"
+    "build": "neutrino build --presets ./neutrino-custom/production.js",
```

Comment:
We'll need to use the relative paths here due to the use of `--no-bin-links` by #2243.
In the case of neutrino iirc that will be `./node_modules/neutrino/bin/neutrino`.